### PR TITLE
Added image-snapshot coverage for Tree

### PR DESCRIPTION
**Pull request by lukelooker**
### :sparkles: Changes

-Add snapshot tests for `Tree`

### :white_check_mark: Requirements

- [ ] Includes test coverage for all changes
- [x] Build and tests are passing
- [x] Update documentation
- [ ] Updated CHANGELOG
- [ ] Checked for i18n impacts
- [ ] Checked for a11y impacts
- [x] PR is ideally < ~400LOC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,13 +62,13 @@ We recommend using [Node Version Manager (NVM)](https://github.com/nvm-sh/nvm#in
 
 #### Storybook
 
-Storybook is a tool we use develop, demonstrate and document components in isolation.
+Storybook is a tool used develop, demonstrate and document components in isolation.
```
Comment (ElliotTheHuman) [inline README.md:65]: nit: Should this be "used to"?

**Review by ElliotTheHuman — COMMENTED** [review]
I _think_ that `packages/components/snapshots/Tree/File Selector-snap.png` is the same as the `File Tree` snapshot.

In addition, not seeing a `FileSelector` component in any of the `Tree`-related stories. Was this just a deprecated name of `FileTree`?

File: README.md (hunk 1 above)
Comment (lukelooker) [reply README.md]: Yep

**Comment by lukelooker** [comment]
> I _think_ that `packages/components/snapshots/Tree/File Selector-snap.png` is the same as the `File Tree` snapshot.
> 
> In addition, not seeing a `FileSelector` component in any of the `Tree`-related story files. Was this just a deprecated name of `FileTree`?

Yep - I renamed the story but image-snapshots doesn't (yet) automatically clean-up unused snapshots.

**Review by ElliotTheHuman — APPROVED** [review]
LGTM!

**Pull request merged**